src/ShowPage/ShowPage.js

```diff
@@ -24,40 +21,48 @@ class ShowPage extends Component {
 
   determineSingleMovie = (id) => {
     getSingleMovie(id)
-    .then(singleMovie => this.setState({currentMovie: singleMovie}))
-    .catch(error => this.setState({ error: error }));
+      .then(singleMovie => this.setState({currentMovie: singleMovie.movie}))
```

Comment:
Glad we caught this, as the fetch request returns a `Promise` in the form of an object, with a `key` of `movie` and a `value` of a nested object (the data we actually want to set into state).